ChangeLog.d/asn1-missing-guard-in-rsa.txt

```diff
@@ -0,0 +1,3 @@
+Bugfix
+   * MBEDTLS_ASN1_PARSE_C and MBEDTLS_ASN1_WRITE_C is automatically enabled
```

Comment:
s/is/are/ for plural

Reply:
And it should make clear that this is the fix, and not the issue, so s/is/are now/